controller/k8s/api.go

```diff
@@ -183,6 +183,8 @@ func (api *API) GetObjects(namespace, restype, name string) ([]runtime.Object, e
 		return api.getRCs(namespace, name)
 	case k8s.Services:
 		return api.getServices(namespace, name)
+	case k8s.Authority:
+		return []runtime.Object{}, nil
```

Comment:
Is this change required for this branch? As far as I can tell, `GetObjects` is only called from the `getKubernetesObjectStats` func in stat_summary.go, and we don't call that method for authorities.